apps/charterafrica/src/payload/fields/ecosystem/airtableColumnSelect.js

```diff
@@ -6,16 +6,8 @@ import {
 } from "payload/components/forms";
 import { select } from "payload/dist/fields/validations";
 import { createElement, useMemo } from "react";
-import useSWR from "swr";
 
-import fetchJson from "../../../utils/fetchJson";
-
-function getUrl(baseId) {
-  if (baseId) {
-    return `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/resources/ecosystem/schema?source=airtable&url=/meta/bases/${baseId}/tables`;
-  }
-  return null;
-}
+import { schema as globalSchema } from "./airtableBaseSelect";
```

Comment:
Rather than importing schema from `airtableBaseSelect` which tightly couples `airtableBaseSelect` to `airtableColumnSelect`, can't we pass `schema` to `airtableColumSelect` similar to how we're passing `tableField`?